README.rst

```diff
@@ -695,11 +689,8 @@ targets that correctly clean the ``runtime`` directory without deleting it, but
 keep this in mind whenever you are not using ``make`` (e.g. running tests from
 your IDE).
 
-Note that the pre-commit checks are currently not working with the default
-Python version. This is because isort dropped Python 3.7 support. You have to
-use at least Python 3.8 for the pre-commit checks. Use ``pipx`` or ``brew`` or
-… to install pre-commit and use the global installation, there is also no
-dependency on it.
+Use ``pipx`` or ``brew`` or to install pre-commit and use the global installation,
```

Comment:
```suggestion
Use ``pipx`` or ``brew`` to install ``pre-commit`` and use the global installation,
```

Reply:
@jjaakola-aiven do you want to fix this before merging?

Reply:
Done.